recipes/cuda-python/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "cuda-python" %}
+{% set version = "11.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/cuda-python/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: e1c1bd49919c8646b7d84e3e63f5a0a971b64a69a9c986194d66f17cd26ee006
+
+build:
+  number: 0
+  skip: true  # [py<37 or osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - setuptools
+    - cython
+    - pycparser
+    - autopxd2
+    - pyclibrary
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - cuda
+
+about:
+  home: https://github.com/NVIDIA/cuda-python
+  license: LicenseRef-LICENSE
```

We might want to clarify this license name a bit more. Made one suggestion below, but would be good to check with others offline if there is a preferred name for this

Also to ensure the license file gets picked up from the source code and included in packages & installs, would suggest adding the `license_file` line below

```suggestion
  license: LicenseRef-NVIDIA-SOFTWARE-LICENSE
  license_file: LICENSE
```